src/styles/main.scss

```diff
@@ -1,3 +1,8 @@
-body {
-  margin: 0;
+* {
+  box-sizing: border-box;
 }
```

Comment:
Non-obligatory changes:

- I think it's better to move this part to `_reset.scss` as you literally doing this by choosing selector `*`.
- You can organize your folder structure by moving helpers (`_reset`, `_variables`, `extends`) to `utils` folder and blocks (`main`, `calendar-block`) to `blocks`.
- Rename `calendar-block` to `calendar`.

Reply:
Thanks!  in previous work I put this part with '*' to reset file , but mentor said  it wasn't  reset style.You are right about the other.